src/commands/logout.js

```diff
@@ -1,12 +1,14 @@
 const { ApifyCommand } = require('../lib/apify_command');
-const { GLOBAL_CONFIGS_FOLDER } = require('../lib/consts');
+const { AUTH_FILE_PATH } = require('../lib/consts');
 const { success } = require('../lib/outputs');
 const { rimrafPromised } = require('../lib/files');
+const { regenerateLocalDistinctId } = require('../lib/telemetry');
 
 class LogoutCommand extends ApifyCommand {
     static async run() {
```

Comment:
There's a bunch of commands using `static async run()` instead of just `async run()`, and for those the `finally()` handler is not called. We should move all of them to just `async run()` to fix it.

Reply:
Good catch